README.md

```diff
@@ -0,0 +1,35 @@
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
```

Comment:
Is this right? It seems to say "Discourse"

Reply:
Yes, like here: https://github.com/canonical/indico-operator
It shows the Discourse status where the docs are.